tests/integration/tier1/destructive/kernel-excluded-in-conf/test_kernel_excluded_in_conf.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from conftest import TEST_VARS
+
+
+@pytest.mark.parametrize("yum_conf_exclude", [["kernel", "kernel-core", "redhat-release-server"]], indirect=True)
+def test_kernel_excluded_in_conf(convert2rhel, yum_conf_exclude):
+    """
+    Verify, the conversion does not raise:
+    'Could not find any kernel from repositories to compare against the loaded kernel.'
+    When `exclude=kernel kernel-core redhat-release-server` is defined in yum.conf
+    Verify IS_LOADED_KERNEL_LATEST has succeeded is raised.
+    Reference ticket: https://issues.redhat.com/browse/RHELC-774
```

Comment:
Love the reference here.
Could be also included in the metadata as 
```
link:
  verifies: https://issues.redhat.com/browse/RHELC-774
```
See https://github.com/oamg/convert2rhel/blob/main/tests/integration/common/checks-after-conversion/test_verify_strings_in_log.py

Reply:
Alright, I put it into the metadata file and also to here https://github.com/oamg/convert2rhel/pull/1346/files#diff-cc849537e44530b2b2d10af1f12d93997ca4371954b354a4104f1639d186da5aR152